### ISSUE TITLE
Menu: label texts and icons are broken

### ISSUE THREAD

**Issue by hramakumar** [eBay/ebayui-core#378]
# Bug Report

## Description
All menu  texts and icons are broken

repro
go to http://ebayuicomponentstestenv.vip.qa.ebay.com/ebayUIStaging/ds4

scroll to the the menu area

## Screenshots
![menutextandicon](https://user-images.githubusercontent.com/13157115/44678442-379a4300-a9ed-11e8-96cd-4206f604f8df.png)

**Comment by yomed** [comment]
@seangates Should we create `chevron-down-bold` and `chevron-up-bold` for ds4 that are just straight copies of `chevron-down` and `chevron-up`?

**Comment by yomed** [comment]
@hramakumar For the text issue, use the `text` attribute instead of `label`.

**Comment by hramakumar** [comment]
@yomed yes the text is not an issue anymore...Lets narrow down the scope of this bug to icons.

**Comment by hramakumar** [comment]
verified
